common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VraNgPackageStore.java

```diff
@@ -324,6 +350,6 @@ protected final VraNgPackageContent getPackageContent(final Package pkg) {
 	@Override
 	protected final void deleteContent(final Content content, final boolean dryrun) {
 		throw new UnsupportedOperationException(
```

Comment:
Just for my info, what is in general the purpose of this function and the expected diff from the deletePackage functions which now have implementation?

Reply:
Deletion is package based, vro and vra had packages that you can import. Those packages have "content" in them that could be deleted partially.

`deleteContent` is a function that need to be implemented in every "PackageStore". Currently the "VRA", "VRO" and "VCD" have an implementation of it.

The function works when you want to delete "old" content only, or "new" without "old" (not 100% sure on this one, don't want to check, but it doesn't matter). It's meant to delete only a subset of the data as for example, when you are deleting the "old" package, it may still contain data from the "newest" package you want to keep.

Reply:
deletePackage will delete the entire package with all of it's content, while deleteContent will delete a single resource from the package.. so One WF, or action or config element, etc

Reply:
oh got it thanks